packages/asset-swapper/src/utils/market_operation_utils/types.ts

```diff
@@ -28,6 +28,8 @@ export enum AggregationError {
 export enum ERC20BridgeSource {
     Native = 'Native',
     Uniswap = 'Uniswap',
+    UniswapV2 = 'UniswapV2',
+    UniswapV2Eth = 'UniswapV2Eth',
```

Comment:
```suggestion
    UniswapV2 = 'Uniswap_V2',
    UniswapV2Eth = 'Uniswap_V2_ETH',
```

I promised Matcha we would stick to this pattern where the first chunk before `_` is the "real" source.

cc @johnrjj